dronegen/tag.go

```diff
@@ -308,6 +319,34 @@ func tagCopyPackageArtifactCommands(b buildType, packageType string) []string {
 	return commands
 }
 
+// createReleaseAssetCommands generates a set of commands to create release & asset in release management service
+func tagCreateReleaseAssetCommands(b buildType) []string {
+	commands := []string{
+		`WORKSPACE_DIR=$${WORKSPACE_DIR:-/}`,
+		`VERSION=$(cat $WORKSPACE_DIR/go/.version.txt)`,
+		fmt.Sprintf(`RELEASES_HOST=%v`, releasesHost),
+		`echo $RELEASES_CERT | base64 -d > $WORKSPACE_DIR/releases.crt`,
+		`echo $RELEASES_KEY | base64 -d > $WORKSPACE_DIR/releases.key`,
+		`trap "rm -f /releases.crt /releases.key" EXIT`,
+		`CREDENTIALS="--cert $WORKSPACE_DIR/releases.crt --key $WORKSPACE_DIR/releases.key"`,
+		`which curl || apk add --no-cache curl`,
+		fmt.Sprintf(`cd $WORKSPACE_DIR/go/artifacts
+for file in $(find . -type f ! -iname '*.sha256'); do
+  # Skip files that are not results of this build
+  # (e.g. tarballs from which OS packages are made)
+  [ -f "$file.sha256" ] || continue
+
+  product="$(basename "$file" | sed -E 's/(-|_)v?[0-9].*//')" # extract part before -vX.Y.Z
```

Comment:
Make sure pattern matches the end of the filename?

```suggestion
  product="$(basename "$file" | sed -E 's/(-|_)v?[0-9].*$//')" # extract part before -vX.Y.Z
```

this is still a bit iffy IMO, but I guess it's alright.